docs/src/components/Card.js

```diff
@@ -15,6 +15,22 @@ type Props = {|
   showHeading?: boolean,
 |};
 
+export const copyPaste = (hash: string): boolean => {
```

Comment:
There isn't any pasting happening, so a better name would probably be something like "copyToClipboard"?

Reply:
indeed... DONE